scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -159,6 +161,21 @@ namespace <%= namespace %> {
         }
 
         /// <summary>
+        /// Launches an external browser with the Cart's web checkout link loaded. This can be used to perform
+        /// a cart checkout when no native web view is available, such as on desktop, or from the Unity editor.
+        /// </summary>
+        /// <param name="success">called when the game has regained focus and the checkout was successful.</param>
+        /// <param name="cancelled">called when the game has regained focus but the checkout has not completed.</param>
+        /// <param name="failure">called when the game has regained focused and an error has occured during checkout.</param>
+        public void CheckoutWithExternalBrowser(CheckoutSuccessCallback success, CheckoutCancelCallback cancelled, CheckoutFailureCallback failure) {
```

Comment:
So if we go options 3, then yeah, we'd definitely need to change `CheckoutWithNativeWebView` to something else, I was thinking`CheckoutWithWebView`.

We'd do something like to pick which WebCheckout implementation to use,

```
#if UNITY_IOS
WebCheckout = new iOSWebCheckout(this, client);
NativeCheckout = new iOSNativeCheckout(State);
#elif UNITY_ANDROID
WebCheckout = new AndroidWebCheckout(this, client);
NativeCheckout = new AndroidNativeCheckout(State);
#elif STANDALONE OR DESKTOP
WebCheckout = new UnityWebCheckout(this, client);;
NativeCheckout = null;
#else
WebCheckout = null;
NativeCheckout = null;
#endif
```

That's actually pretty elegant. @dengjeffrey you cool with that too? cc @sleroux